bin/prettier.js

```diff
@@ -368,6 +377,24 @@ function writeOutput(result) {
 }
 
 function eachFilename(patterns, callback) {
+  // The ignorer will be used to filter file paths after the glob is checked,
+  // before any files are actually read
+  const ignoreFilePath = options.ignorePath || DEFAULT_IGNORE_FILENAME;
+  const absoluteIgnoreFilePath = path.isAbsolute(ignoreFilePath)
```

Comment:
I think this can just be:

```js
const ignoreFilePath = path.resolve(options.ignorePath);
```

You don't need to explicitly pass `process.cwd()`.